tests/WE2E/end_to_end_tests.sh

```diff
@@ -70,18 +82,17 @@ sed -i 's|EXECDIR="${SR_WX_APP_TOP_DIR}/bin"|EXECDIR="${SR_WX_APP_TOP_DIR}/bin_i
 #-----------------------------------------------------------------------
 
 # Load Python Modules
-source ${APP_DIR}/env/wflow_${machine}.env
 cmd="${APP_DIR}/env/wflow_${machine}.env"
+source ${cmd}
 
-echo "-- Python command =>"  $cmd
+echo "-- Load environment =>"  $cmd
 
 # If experiments list file exists, remove it, and add the experiemnts to a new expts list file
-if [ -f "auto_expts_list.txt" ] ; then
-  rm "auto_expts_list.txt"
-fi
-echo ${expts} > auto_expts_list.txt
+auto_file="auto_expts_list.txt"
+
+rm -rf ${auto_file}
```

Comment:
We can use flags on `rm` that don't require checking for the existence of the file -- one less file system check, which can be expensive.

Reply:
Agree. Don't know why I didn't simplify this to `rm -rf`.